### PR TITLE
set correct page title

### PR DESCRIPTION
**Pull request by swalke-ch**
set correct page title & fix spelling mistakes in file name

### CODE REVIEW

**Review by abhiahirv1 — APPROVED** [review]

File: src/routers/sectorYouWorkInRouter.ts
Comment (ttingle-ch) [inline src/routers/sectorYouWorkInRouter.ts:1]: You need to add the title to the post router too

Comment (swalke-ch) [reply src/routers/sectorYouWorkInRouter.ts:1]: done

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Review by abhiahirv1 — APPROVED** [review]

**Comment by swalke-ch** [comment]
This is been done as part of other PR

**Pull request closed without merging**